jetty-servlets/src/main/java/org/eclipse/jetty/servlets/CrossOriginFilter.java

```diff
@@ -159,7 +159,8 @@
     private boolean anyHeadersAllowed;
     private List<String> allowedOrigins = new ArrayList<String>();
     private List<String> allowedTimingOrigins = new ArrayList<String>();
-    private List<String> allowedMethods = new ArrayList<String>();
+    private String  allowedMethodsCommify;
```

Comment:
What does commify mean?

Reply:
There is a method called CrossOriginFilter.commify that was called for each request. This variable holds the `commified` version :-). An alternative could be to call it  allowMethodsValue (to point that will be hold the value of the header Access-Control-Allow-Methods). What do you think?